scaleway/helpers.go

```diff
@@ -696,6 +697,19 @@ func diffSuppressFuncTimeRFC3339(k, oldValue, newValue string, d *schema.Resourc
 	return t1.Equal(t2)
 }
 
+func diffSuppressFuncBase64(k string, oldValue string, newValue string, d *schema.ResourceData) bool {
+	if oldValue == newValue {
+		return true
+	}
+
+	oldStr, err1 := base64.StdEncoding.DecodeString(oldValue)
+	newStr, err2 := base64.StdEncoding.DecodeString(newValue)
+	if err1 != nil || err2 != nil {
+		return false
+	}
+	return string(oldStr) == string(newStr)
```

Comment:
```suggestion
	return oldValue == newValue
```

Two encoding of the same value outputs the same base64, why do we need the decoding part?